datasets/pyproject.toml

```diff
@@ -0,0 +1,150 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "flwr-datasets"
+version = "0.0.1"
+description = "Flower Datasets"
+license = "Apache-2.0"
+authors = ["The Flower Authors <hello@flower.dev>"]
+readme = "README.md"
+homepage = "https://flower.dev"
+repository = "https://github.com/adap/flower/datasets"
+documentation = "https://flower.dev/docs/datasets"
+keywords = [
+    "flower",
+    "fl",
+    "federated learning",
+    "federated analytics",
+    "federated evaluation",
+    "machine learning",
+    "dataset",
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
```

Comment:
We shouldn't declare this as stable yet

Reply:
Changed to: Development Status :: 1 - Planning